### ISSUE TITLE
types_naming, false positive, ENUM

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1911]
```abap
TYPES: BEGIN OF ENUM ty_category BASE TYPE string,
         hello,
         world,
       END OF ENUM ty_category.
```

**Comment by larshp** [comment]
hmm

**Comment by larshp** [comment]
this might have been fixed, closing
